engine/src/main/java/org/kigalisim/engine/state/EngineConstants.java

```diff
@@ -17,6 +17,13 @@
  *
  * <p>Defines context levels for variable scoping and base units for different
  * stream types in the simulation engine.</p>
+ * 
+ * <p>Key differences from JavaScript engine_const.js:
```

Comment:
Thank you for this! However, I think I will document this elsewhere. Please remove this new paragraph (<p>)